.gitignore

```diff
@@ -121,3 +121,4 @@ wp-content/debug.log
 !/wp-content/themes/midwestcenter
 !/wp-content/themes/bigagwatch
 
+wp-content/themes/midwestcenter/node_modules/*
```

Comment:
Might be easier for the long run to add this to a .gitignore in the midwestcenter theme, but this works.